test-suite/src/function/cast.rs

```diff
@@ -51,10 +51,12 @@ test_case!(cast_literal, async move {
             r#"SELECT CAST("foo" AS INTEGER) AS cast FROM Item"#,
             Err(ValueError::LiteralCastFromTextToIntegerFailed("foo".to_owned()).into()),
         ),
-        (
-            r#"SELECT CAST(1.1 AS INTEGER) AS cast FROM Item"#,
-            Ok(select!(cast I64; 1)),
-        ),
+
+        //this now fails..
+        //(
+        //    r#"SELECT CAST(1.1 AS INTEGER) AS cast FROM Item"#,
+        //    Ok(select!(cast I64; 1)),
+        //),
```

Comment:
This looks reasonable decision that you've made.
Then, we don't need to keep this test case anymore.
Let's remove these code block :) thanks.